client_internals_it_test.go

```diff
@@ -387,9 +387,9 @@ func TestClientInternal_EncodeData(t *testing.T) {
 func TestProxyManagerShutdown(t *testing.T) {
 	clientTester(t, func(t *testing.T, smart bool) {
 		ctx := context.Background()
-		tc := it.StartNewClusterWithOptions("proxy-manager-graceful-shutdown", 5701, 1)
+		tc := it.StartNewClusterWithOptions("proxy-manager-graceful-shutdown", 35701, 1)
```

Comment:
Could you use `t.Name()` as the cluster name instead of `proxy-manager-graceful-shutdown` ?